core/src/com/unciv/ui/options/MultiplayerTab.kt

```diff
@@ -134,19 +170,86 @@ private fun fixTextFieldUrlOnType(TextField: TextField) {
     }
 }
 
-private fun addMultiplayerTurnCheckerDelayBox(table: Table, settings: GameSettings) {
-    table.add("Time between turn checks out-of-game (in minutes)".toLabel()).left().fillX()
+private class RefreshOptions(val delay: Duration, val label: String) {
+    override fun toString(): String = label
+    override fun equals(other: Any?): Boolean = other is RefreshOptions && delay == other.delay
+    override fun hashCode(): Int = delay.hashCode()
+}
+
+
+private val curRefreshDropboxOptions =
+    (listOf<Long>(10, 20, 30, 60).map { RefreshOptions(Duration.ofSeconds(it), "$it " + "Seconds".tr()) }).toGdxArray()
+
+private val curRefreshCustomServerOptions =
+    (listOf<Long>(3, 5).map { RefreshOptions(Duration.ofSeconds(it), "$it " + "Seconds".tr()) } + curRefreshDropboxOptions).toGdxArray()
+
+private val allRefreshDropboxOptions =
+    (listOf<Long>(1, 2, 5, 15).map { RefreshOptions(Duration.ofMinutes(it), "$it " + "Minutes".tr()) }).toGdxArray()
+
+private val allRefreshCustomServerOptions =
+    (listOf<Long>(15, 30).map { RefreshOptions(Duration.ofSeconds(it), "$it " + "Seconds".tr()) } + allRefreshDropboxOptions).toGdxArray()
+
+private val turnCheckerDropboxOptions =
+    (listOf<Long>(1, 2, 5, 15).map { RefreshOptions(Duration.ofMinutes(it), "$it " + "Minutes".tr()) }).toGdxArray()
+
+private val turnCheckerCustomServerOptions =
+    (listOf<Long>(30).map { RefreshOptions(Duration.ofSeconds(it), "$it " + "Seconds".tr()) } + allRefreshDropboxOptions).toGdxArray()
+
+private fun <T> List<T>.toGdxArray(): Array<T> {
+    val arr = Array<T>(size)
+    for (it in this) {
+        arr.add(it)
+    }
+    return arr
+}
+
+private fun usesDropbox(settings: GameSettings) = settings.multiplayer.server == Constants.dropboxMultiplayerServer
+
+private fun addRefreshSelect(
```

Comment:
This leaves two cells in destination `table`, but calling code doesn't 'grok' there's now two columns where there was one, and other controls on the page lack the colspan... Noo biggie